sass/includes/_highlight-gallery.scss

```
@@ -76,7 +76,7 @@
 
     &:hover,
     &:visited {
-      color: $color__grey-500;
+      color: $color__white;
 
       svg {
         fill: $color__grey-500;
```

```suggestion
        fill: $color__white;
```

If we're updating the text, we should update the fill colour for the chevron icon too

---

Now that we're changing the link to always be white, we don't have any change on interaction / hover - I wonder if it's worth adding an underline on hover to distinguish the two. Perhaps we should leave this for now and pick it up with Nick when he gets back - I can't see any explicit designs for interaction states on these links at the mo so possibly something we haven't considered yet.